domains/stephenvs.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "Portfolio site for Stephen Van Sittert",
+  "owner": {
+    "username": "Stephenvs123",
+    "email": "berith15@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://portfolio.steve-home-lab.com/"
```

CNAME can't contain protocol or `/`
also cannot access website (Sorry, you have been blocked)
```suggestion
    "CNAME": "portfolio.steve-home-lab.com"
```